src/main/java/org/myrobotlab/jme3/MainMenuState.java

```diff
@@ -37,7 +37,7 @@
   transient JMonkeyEngine jme = null;
   transient Jme3App app;
   transient Container main;
-  transient Label breadCrumbs;
+  transient Label breadCrumbsx;
```

Comment:
Is this a typo? did you really want to rename this? or maybe you want to delete this variable?